apps/storybook/src/HeatmapMesh.stories.tsx

```diff
@@ -0,0 +1,121 @@
+import {
+  getDomain,
+  getMockDataArray,
+  HeatmapMesh,
+  VisCanvas,
+} from '@h5web/lib';
+import type { HeatmapMeshProps, Domain } from '@h5web/lib';
+import { ScaleType, toTypedNdArray } from '@h5web/shared';
+import type { Meta, Story } from '@storybook/react/types-6-0';
+import ndarray from 'ndarray';
+import {
+  ByteType,
+  FloatType,
+  HalfFloatType,
+  IntType,
+  LinearFilter,
+  NearestFilter,
+  ShortType,
+  UnsignedByteType,
+  UnsignedIntType,
+  UnsignedShortType,
+} from 'three';
+
+import FillHeight from './decorators/FillHeight';
+
+const dataArray = getMockDataArray('/nD_datasets/twoD');
+const domain = getDomain(dataArray.data);
+const [rows, cols] = dataArray.shape;
+
+const uint16DataArray = ndarray(
+  Uint16Array.from([0x49_00, 0x4d_00, 0x4f_80, 0x51_00]),
+  [2, 2]
+);
+const uint16Domain: Domain = [10, 40];
+
+const Template: Story<HeatmapMeshProps> = (args) => {
+  return (
+    <VisCanvas
+      abscissaConfig={{ visDomain: [0, cols] }}
+      ordinateConfig={{ visDomain: [0, rows] }}
```

Comment:
Given that you set different arrays for `values`, it might be better to get `cols` and `rows` from `args.values` in here

Reply:
Ha, good catch, didn't even notice that the axes were wrong.

Reply:
I've also set `isIndexAxis` for obvious reasons.